#!/usr/bin/env python
PACKAGE = "ork_morse_interface"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("id_mug",    str_t,    0, "A string parameter of the ORK_MORSE interface corresponding to the ID of the Mug",  "MUG")
gen.add("id_medic",    str_t,    0, "A string parameter of the ORK_MORSE interface corresponding to the ID of the Medic box",  "MEDIC")
gen.add("id_penholder",    str_t,    0, "A string parameter of the ORK_MORSE interface corresponding to the ID of the penHolder",  "PENHODLER")
gen.add("start_timeout",    int_t,    0, "An Integer parameter corresponding to the time, in number of ros::loop, that the object will be displayed within morse. (current ros::loop is 100Hz -> startTimeout=100 means 1 sec etc...)", 200,  0, 1000)

exit(gen.generate(PACKAGE, "ork_morse_interface", "ORKMorse"))
